docs/core/usage.qmd

```diff
@@ -162,6 +162,7 @@ between nodes, and never have storage of their own.
 column        | type     | restriction
 --------------| -------- | -----------
 fid           | Int      | unique, sorted
+node_id       | Int      | -
 type          | String   | known node type
```

Comment:
```suggestion
node_id       | Int      | sorted
node_type          | String   | known node type
```

The `fid` is needed by GeoPackage, but not really part of our datamodel, right? And GeoPandas adds it for us. So perhaps better to leave out?

Since you added `node_id`, not `id` to the Node table, perhaps we should rename `type` to `node_type` for consistency. It is slightly redundant since the table is also called Node, but it is better to always use the same name. That way it is also easier to join columns between different tables.

Reply:
Fine by me!